src/helpdesk_app/urls.py

```diff
@@ -19,6 +19,9 @@
 
 urlpatterns = [
     path('', lambda request: redirect('search/', permanent=False), name='home'),
+    path('en/search/thumbsUpClicked', views.thumbsUpClicked, name='thumbsUpClicked'),
```

Comment:
I think there's a problem with specifying these URLs with `en/`. If you change your localization to Spanish, then run a search and then thumbs up something, you can see that it doesn't +1.

I'm not sure if this would work, but in `helpdesk_proj/urls.py` the `search` path is defined in `i18n_patterns.` This automatically generates the `en/` and `es/` equivalent for `search`. Try adding in `search/resourceClicked` and the rest to that table and hopefully it'll work?

Reply:
Ah yes, that was confusing me, but thanks this works!